NotificationApi/NotificationApi.IntegrationTests/Steps/HearingAmendmentSteps.cs

```diff
@@ -41,6 +41,18 @@ public void GivenIHaveAHearingAmendmentForAnEjudJudgeEmailNotificationRequest()
             InitCreateNotificationRequest(request, _context);
         }
         
+        [Given(@"I have a hearing amendment for an ejud joh email notification request")]
+        public void GivenIHaveAHearingAmendmentForAnEjudJohEmailNotificationRequest()
+        {
+            var messageType = MessageType.Email;
+            var templateType = NotificationType.HearingAmendmentEJudJoh;
+            var parameters = InitGenericAmendmentParams();
+            parameters.Add("judge", $"{Faker.Name.FullName()}");
```

Comment:
You have the wrong param here. this is for the JOH, not a judge Copy and paste?

Reply:
> You have the wrong param here. this is for the JOH, not a judge Copy and paste?

doh, curse of copy-paste it is fixed now